### DISCUSSION TITLE
How can I hide a hidden true component in Datagrid when Initialize Empty true?

### DISCUSSION THREAD

**Discussion by Dev-Trilok** [formio/formio.js#4996] [Q&A]
I want to add a hidden text field in Datagrid with the Initialize Empty bit true. On form render, I do want not to show the Hidden field.
How can I hide it?

I have added a grid with hidden field.
![image](https://user-images.githubusercontent.com/48346388/209297031-832f8303-8f08-4642-bcbb-17186467b5e2.png)

When I set Datagrid Iniliaze Empty and render form it will show me like the below image (show hidden column header name or column )

![image](https://user-images.githubusercontent.com/48346388/209297260-32591301-11da-4eb0-b51e-430c60ccbd1c.png)

But I want remove the hidden column on the form render like

![image](https://user-images.githubusercontent.com/48346388/209297411-3de5ceb5-6367-4582-ace7-b19f38bed5f3.png)

**Comment by lakim** [comment]
@Dev-Trilok did you eventually find a solution?
